src/Common/PoolWithFailoverBase.h

```diff
@@ -111,17 +111,16 @@ class PoolWithFailoverBase : private boost::noncopyable
             size_t min_entries, size_t max_entries, size_t max_tries,
             const TryGetEntryFunc & try_get_entry,
             const GetPriorityFunc & get_priority = GetPriorityFunc(),
-            bool fallback_to_stale_replicas = true);
-
-    void reportError(const Entry & entry);
+            bool fallback_to_stale_replicas = true,
+            size_t error_ignore = 0);
```

Comment:
Do you mind making them explicit? We're constantly finding forgotten default parameters all over the code. IIRC we even had this exact problem when we added "distributed_errors_cap" to the constructor of this class.

Reply:
>Do you mind making them explicit? We're constantly finding forgotten default parameters all over the code.

Great idea, initially it was added just like `fallback_to_stale_replicas` (with default value)

Reply:
Done (plus some other cleanup)

P.S. I'm trying to keep PRs without cleanups/refactoring to reduce number of changes (lots of changes is hard to review), but looks like it is not a problem for you, so I will consider deviating from this rule in some cases